addon-test-support/-private/execution_context.js

```diff
@@ -28,10 +28,14 @@ export function getExecutionContext(pageObjectNode) {
   let contextName;
   if (integrationTestContext) {
     contextName = 'integration';
-  } else if (isRfc268Test()) {
-    contextName = 'rfc268';
-  } else {
+  } else if (isAcceptanceTest()) {
     contextName = 'acceptance';
+  } else if (supportsRfc268()) {
+    contextName = 'rfc268';
+  }
+
+  if (!contextName) {
+    throw new Error('Can not detect test type. Please make sure you use the latest version of "@ember/test-helpers".');
```

Comment:
needs coverage

Reply:
unfortunatelly, in added test, there are some `require.unsee` conflicts with https://github.com/san650/ember-cli-page-object/blob/master/tests/unit/-private/compatibility-test.js, which I could not figure out.
Removing a new test :(